src/tests/ftest/daos_test/daos_core_test_dfs.yaml

```diff
@@ -11,11 +11,11 @@ hosts:
     - client-F
     - client-G
     - client-H
-
-timeout: 250
+timeout: 300
 pool:
   #This will create 8G of SCM and 16G of NVMe size of pool.
   scm_size: 8G
+  nvme_size: 16G
```

Comment:
Minor: Why is this needed? For daos_test, nvme size was automatically created based on size of scm if I remember correctly. Is it different for dfs_test now?

Reply:
hmm no, i don't know where i copied that from. I will remove it when i add more tests in a future patch.